lib/spack/spack/build_systems/python.py

```diff
@@ -258,7 +252,7 @@ def install_args(self, spec, prefix):
                  '--install-purelib=%s' % pure_site_packages_dir,
                  '--install-platlib=%s' % plat_site_packages_dir,
                  '--install-scripts=bin',
-                 '--install-data=""',
+                 '--install-data=',
```

Comment:
Why this change? This seems to be the only diff to upstream: https://github.com/BlueBrain/spack/compare/hotfix-python-install..spack:releases/v0.16#diff-61e125b04c4e9783dda6c50777b7a9ccb298b342a1275a6df8fd736a1f5dc829

Reply:
Because on at least my shell, this gives a literal `""/share/…` in the install prefix instead of just `share/…`

Reply:
See also: https://github.com/spack/spack/blob/develop/lib/spack/spack/build_systems/python.py#L266

Reply:
OK!